packages/kbn-i18n/src/server/ui_i18n_mixin.js

```diff
@@ -17,52 +17,47 @@
  * under the License.
  */
 
-import { defaults, compact } from 'lodash';
-import langParser from 'accept-language-parser';
-
-import { I18n } from './i18n';
+/**
+ @typedef Messages - messages tree, where leafs are translated strings
```

Comment:
__question:__ is there any reason why we can't leave `ui_mixin` in Kibana as it was before? Even though we don't import from Kibana here, we still depend on it explicitly via `kbnServer & friends` arguments.

Reply:
Just wanted to encapsulate all i18n logic into a separate package. Ok, I will move this file back to `ui_i18n` folder